unittests/src/main/java/uk/ac/manchester/tornado/unittests/common/TornadoTestBase.java

```diff
@@ -93,4 +93,11 @@ public void checkForPTX() {
         }
     }
 
+    public void checkForOpenCL() {
+        int driverIndex = TornadoRuntime.getTornadoRuntime().getDefaultDevice().getDriverIndex();
+        if ("OpenCL".equals(TornadoRuntime.getTornadoRuntime().getDriver(driverIndex).getName())) {
```

Comment:
I suggest adding the method `getBackendType`  

`TornadoRuntime.getTornadoRuntime().getBackendType(driverIndex)`

This returns a value based on an enum:

`OCL, PTX, etc.` 

And we avoid String comparisons.